src/cs50/sql.py

```diff
@@ -78,13 +50,25 @@ def connect(dbapi_connection, connection_record):
         # Register listener
         sqlalchemy.event.listen(self._engine, "connect", connect)
 
-        # Log statements to standard error
-        logging.basicConfig(level=logging.DEBUG)
+        # Configure logger
+        self._logger = logging.getLogger(__name__)
```

Comment:
How would a user disable this logged if it's not a named logger, as via `logging.getLogger("cs50")`?

Reply:
```
logging.getLogger('cs50.sql').disabled = True
```

Reply:
Hm, thoughts on promoting back to `logging.getLogger('cs50')`, since if we add other diagnostics, they might as well all live together for simplicity?

Reply:
I honestly think this is better for modularity in case you want to disable some CS50 loggers by not all. If we add more modules and more loggers in the future, we can factor this logic out and have a parent `cs50` logger and have child loggers e.g., `cs50.foo`, `cs50.bar`, etc with no handlers pass messages to the parent to handle. Then you'd be able to disable only some loggers if you want or disable all of them by disabling the parent logger (i.e., `cs50`). Happy to revert if you still think the other approach is better!